### PR TITLE
chore(deps): Update Sass, SLDS and related deps

### PR DESCRIPTION
**Pull request by Dottenpixel**
- copy edits
- remove ruby for dart Sass
- update gulp-sass
- update gulp-load-plugins
- update slds in the lock file
- Closes salesforce-ux/design-system-tooling#1505
- Closes [W-10162213](https://gus.lightning.force.com/a07EE00000O84tXYAR)

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -14,9 +14,15 @@ const gulp = require('gulp')
 const del = require('del')
 const browserSync = require('browser-sync')
 const JSON5 = require('json5')
-const gulpLoadPlugins = require('gulp-load-plugins')
 
-const $ = gulpLoadPlugins()
+const $ = require('gulp-load-plugins')({
+  // var sass = require('gulp-sass')(require('sass'));
```
Comment (ambreeen) [inline gulpfile.js:19]: should we remove this comment?

Comment (Dottenpixel) [reply gulpfile.js]: definitely should, all set!

**Review by ambreeen — APPROVED** [review]
🚢

**Review by fredboyle — APPROVED** [review]
🚀

**Pull request merged**